pandas/io/json.py

```diff
@@ -884,3 +939,174 @@ def _recursive_extract(data, path, seen_meta, level=0):
         result[k] = np.array(v).repeat(lengths)
 
     return result
+
+
+# ---------------------------------------------------------------------
+# JSON-Table Schema routines
+# http://specs.frictionlessdata.io/json-table-schema/
+
+
+def as_json_table_type(x):
+    """
+    Convert a NumPy / pandas type to its corresponding json_table.
+
+    Parameters
+    ----------
+    x : array or dtype
+
+    Returns
+    -------
+    t : str
+        the Table Schema data types
+
+    Notes
+    -----
+    This table shows the relationship between NumPy / pandas dtypes,
+    and Table Schema dtypes.
+
+    ==============  =================
+    Pandas type     Table Schema type
+    ==============  =================
+    int64           integer
+    float64         number
+    bool            boolean
+    datetime64[ns]  date
+    timedelta64[ns] duration
+    object          str
```

Comment:
u can infer_dtype in ['string', 'unicode']

thought is_string_dtype wasn't more strict (and didntjis inferring)